### PR TITLE
Edit Breadcumb

### PR DESCRIPTION
**Pull request by lannp**

### CODE REVIEW

**Comment by vietanhbkaaa** [comment]
OK

**Comment by hellovietnam93** [comment]
Ok

File: lib/rails_admin/base.rb [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        end
+
+        register_instance_option :breadcrumb_parent do
+          case
+          when root?
+            [:dashboard]
+          when collection?
+            [:index, bindings[:abstract_model]]
+          when member?
+            if bindings[:abstract_model].to_s == "Course"
```
Comment (dieunb) [inline lib/rails_admin/base.rb]: Course.class_name

File: lib/rails_admin/base.rb [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        end
+
+        register_instance_option :breadcrumb_parent do
+          case
+          when root?
+            [:dashboard]
+          when collection?
+            [:index, bindings[:abstract_model]]
+          when member?
+            if bindings[:object].class.to_s == "Course"
```
Comment (dieunb) [inline lib/rails_admin/base.rb]: `Course.name`

**Pull request merged**